### DISCUSSION TITLE
A923EBE7-151C-8A23-EAF4-72FE40543B75

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#166] [General]
# A923EBE7-151C-8A23-EAF4-72FE40543B75

Aqua is a NASA scientific research satellite in orbit around the Earth, studying the precipitation, evaporation, and cycling of water. It is the second major component of the Earth Observing System (EOS) preceded by Terra (launched 1999) and followed by Aura (launched 2004). Aqua is still operating in 2022.

https://celestia.mobi/addon?item=A923EBE7-151C-8A23-EAF4-72FE40543B75

**Comment by DoojayFrom** [comment]
It’s cool
